crates/lunatic-control-submillisecond/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "lunatic-control-submillisecond"
+version = "0.12.0"
+edition = "2021"
+description = "TBD"
+homepage = "https://lunatic.solutions"
+repository = "https://github.com/lunatic-solutions/lunatic/tree/main/crates"
+license = "Apache-2.0/MIT"
+
+[dependencies]
+lunatic-control = { workspace = true }
+
+anyhow = { workspace = true }
+base64-url = "1.4.13"
+bincode = { workspace = true }
+chrono = { version = "0.4.23", default-features = false, features = [
+  "clock",
+  "serde",
+  "std",
+] }
+getrandom = "0.2.8"
+lunatic = { path = "../../../lunatic-rs", features = ["sqlite"] }
```

Comment:
Can we change these local paths to git urls until everything is published on crates.

Reply:
Changed it all to git urls now, and added instructions to PR description